translate/src/modules/machinery/components/Machinery.tsx

```diff
@@ -80,10 +77,6 @@ export const Machinery = ({
     if (searchInput.current) searchMachinery(searchInput.current.value);
   };
 
-  if (!locale) {
-    return null;
-  }
```

Comment:
I could not find a reason why this component -- which otherwise does not use it -- should not render for a theoretical empty `locale`. As it's now coming from a Context, it'll never be empty.